testCoverage.sh

```diff
@@ -2,11 +2,15 @@
 
 set -e
 echo "" > coverage.txt
+echo "" > dirlist.out
 
-for d in $(find . -name \*.go | xargs -n 1 dirname | uniq | grep -v /vendor/ | grep -v /fuse); do
+find . -name \*.go | xargs -n 1 dirname | uniq | grep -v /vendor/ | grep -v /fuse > dirlist.out
+
+for d in $(sort dirlist.out | uniq); do
```

Comment:
curious - why this change?

Reply:
We have always been testing packages during the coverage step multiple times because we are not uniq'ing between for loop iterations

Reply:
gotcha - nice catch